python-opentimestamps/opentimestamps/calendar.py

```diff
@@ -70,6 +71,14 @@ def submit(self, digest):
             ctx = BytesDeserializationContext(resp_bytes)
             return Timestamp.deserialize(ctx, digest)
 
+    def submit_async(self, msg, q):
+        t = threading.Thread(target=self._submit_async, args=(msg, q))
+        t.start()
+
+    def _submit_async(self, msg, q):
+        calendar_timestamp = self.submit(msg)
+        q.put(calendar_timestamp)
+
```

Comment:
Oh, actually, so let's put this code in the OpenTimestamps Client, rather than this library.

Also, remember that anything under the python-opentimestamps directory is a separate repo; changes should be made in the python-opentimestamps repo rather than the copy within the client repo.

Finally, when you make this change, you don't need to open a new pull-req: just force push the changes with the -f option to git push.